test/misc-tools.js

```diff
@@ -58,7 +58,7 @@ describe('Tools', () => {
   it('should truncate many things!', () => {
     const tests = [
       { value: null, length: 5, suffix: undefined, expected: '' },
-      { value: undefined, length: 5, suffix: '...', expected: '' },
+      { value: undefined, length: 5, suffix: '...', expected: undefined },
```

Comment:
Why change undefined to return as undefined instead empty string?

Reply:
see [here](https://github.com/zapier/zapier-platform-core/pull/151#discussion_r277093772) - I changed that method.

Because we're doing 

```js
data.response_headers = someFunc(data.response_headers)
```

`data` goes from `{a: 1}` to `{a: 1, response_headers: undefined}`. So, I changed that behavior. avoiding `truncate` turn `undefined` into `''` (which felt unintuitive).

The other option would be only call `someFunc` when `data.response_headers` already exists to avoid adding new properties. This felt cleaner.

Reply:
Got it. I wasn't making the connection to it adding a prop in a caller. Makes sense!